lib/google-authenticator-rails/active_record/helpers.rb

```diff
@@ -6,12 +6,18 @@ def set_google_secret
         save
       end
 
-      def google_authentic?(code)
-        GoogleAuthenticatorRails.valid?(code, google_secret_value, self.class.google_drift)
+      def google_authentic?(code,drift=nil)
+        drift = self.class.google_drift if drift.nil?
+        GoogleAuthenticatorRails.valid?(code, google_secret_value, drift)
       end
 
-      def google_qr_uri
-        GoogleQR.new(:data => ROTP::TOTP.new(google_secret_value, :issuer => google_issuer).provisioning_uri(google_label), :size => "200x200").to_s
+      def google_qr_uri(w=200,h=200)
+        GoogleQR.new(:data => ROTP::TOTP.new(google_secret_value, :issuer => google_issuer).provisioning_uri(google_label.to_s), :size => "#{w}x#{h}").to_s
+      end
+
+      def qr_code_png(size=200,level=:h)
```

Comment:
Kina nit-picky, but could you put spaces between `=` and the args? So: `qr_code_png(size = 200, level = :h)`, and maybe a comment on this function as to what those are?